bin/k-secret

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+name="$1"
+subpath="$2"
+
+secret_as_json=$(kubectl get --output json secret "$name")
+
+if [[ "$subpath" = "" ]]; then
```

Comment:
```suggestion
if [[ -z "$subpath" ]]; then
```

Reply:
The way I have it is a Tannerism -- an equality comparison is easier to remember and understand than the mysterious "`-z`"